projects/vdk-plugins/quickstart-vdk/.plugin-ci.yml

```diff
@@ -35,6 +35,10 @@ build-py311-quickstart-vdk:
   extends: .build-quickstart-vdk
   image: "python:3.11"
```

Comment:
I think we should remove 3.11 tests? We want to run tests with only the first and latest support python version